src/test/netbase_tests.cpp

```diff
@@ -14,37 +14,54 @@ using namespace std;
 
 BOOST_FIXTURE_TEST_SUITE(netbase_tests, BasicTestingSetup)
 
+static CNetAddr ResolveIP(const char* ip)
```

Comment:
More helpers! Specifically, this code is identical to the helpers in `net_tests.cpp`. Why aren't these in one place, in a place where they can be used by more than just the tests? Again, this is a backport PR, so it's beyond our scope.